packages/eslint-config-shakacode/rules/javascript.yml

```diff
@@ -17,3 +17,18 @@ rules:
     - 2
     - args: "all"
       argsIgnorePattern: ^_
+
+  # http://eslint.org/docs/rules/max-len
+  # https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L128
+  # Almost identical to airbnb's version, but adds ignore of `import` statements
+  max-len:
+    - 2
+    -
+      code: 100
+      tabWidth: 2
+      ignoreUrls: true
+      ignoreComments: false
+      ignoreRegExpLiterals: true
+      ignoreStrings: true
+      ignoreTemplateLiterals: true
+      ignorePattern: ^import\\s+
```

Comment:
In that example, I agree with you that the top is better. I dunno I guess I'm okay with it then